qir/qat/version.cmake

```diff
@@ -0,0 +1,128 @@
+function (generate_version_header)
+
+	execute_process(COMMAND git describe --dirty=-wip  --always
+	                WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+	                OUTPUT_VARIABLE MICROSOFT_VERSION_FULL
+	                OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+
+    string(REGEX
```

Comment:
The `execute_process` (and L4 - L6) is indented with tab.
The L9+ with spaces

Reply:
That should spaces.